docs/docs/troubleshooting.md

```diff
@@ -26,6 +26,36 @@ Variations of the warnings shown below occur when flashing the `<firmware>.uf2`
 An error along the lines of `CMake Error at (zmk directory)/zephyr/cmake/generic_toolchain.cmake:64 (include): include could not find load file:` during firmware compilation indicates that the Zephyr Environment Variables are not properly defined.
 For more information, click [here](../docs/development/setup.md#environment-variables).
 
+### West build errors
+
+West build errors usually indicates syntax problems in the `<shield>.keymap` file in the compilation process. These errors
+
+:::note
+If you are reviewing these errors in the GitHub Actions tab, they can be found in the `West Build` step of the build process.
+:::
+
+#### devicetree error
+
+A `devicetree error` followed by a reference to the line number on `<shield>.keymap` refers to an issue at the exact line position in that file. For example, this indicates a missing `;`:
```

Comment:
There is some repetition here that can be improved, e.g. the sentence after the code block can be removed if this sentence is worded as below:
```suggestion
A `devicetree error` followed by a reference to the line number on `<shield>.keymap` refers to an issue at the exact line position in that file. For example, below error message indicates a missing `;`at line 109 of the `cradio.keymap` file:
```